packages/gatsby-telemetry/src/store.js

```diff
@@ -11,9 +11,21 @@ const { ensureDirSync } = require(`fs-extra`)
 const Configstore = require(`configstore`)
 
 module.exports = class Store {
-  config = new Configstore(`gatsby`, {}, { globalConfigPath: true })
+  config
 
-  constructor() {}
+  constructor() {
+    try {
+      this.config = new Configstore(`gatsby`, {}, { globalConfigPath: true })
+    } catch (e) {
+      // in case of some permission issues the configstore throws
+      // so fallback to a simple in memory config
+      this.config = {
+        get: key => this.config[key],
+        set: (key, value) => (this.config[key] = value),
```

Comment:
Are these chainable? (i.e. in configstore)

Reply:
It doesn't appear to be so.

Reply:
Then I think this seems reasonable. We _have_ to fall back to something, and if we don't have writable access, this seems OK?